terminal/src/patch.rs

```diff
@@ -32,6 +32,15 @@ impl Comment {
 
         comment.to_owned()
     }
+
+    pub fn append(&self, arg: &str) -> Comment {
```

Comment:
Since you're returning a new comment here, I'd have this function move `self` instead of taking it by reference. This will also avoid others using this function like `comment.append("...");` and not understanding why nothing was appended.

Reply:
I simplified the code by making append mutate self in place of returning a new copy.